sendCustomMessage.js

```diff
@@ -7,9 +8,13 @@ const go = async () => {
   await rtm.start()
   await web.chat.postMessage({
     text: args.m,
-    channel: args.c
+    channel: args.c,
+    as_user: false,
```

Comment:
might be worth abstracting this postmessage call with its new defaults into a new function so we dont have to remember to add `false` every time, but can do that later